bolt-http4k/src/main/java/test_locally/Http4kSlackAppServer.java

```diff
@@ -0,0 +1,50 @@
+package test_locally;
+
+import com.slack.api.bolt.App;
+import org.http4k.server.Http4kServer;
+import org.http4k.server.ServerConfig;
+import org.jetbrains.annotations.NotNull;
+
+import static org.http4k.server.Http4kServerKt.asServer;
+
+/**
+ * Simple Slack App Server wrapper class for Http4k apps.
+ */
+public class Http4kSlackAppServer implements Http4kServer {
+    private final Http4kServer server;
+
+    public Http4kSlackAppServer(App apiApp, ServerConfig serverConfig) {
+        server = asServer(new Http4kSlackApp(apiApp), serverConfig);
+    }
+
+    @NotNull
+    public Http4kServer start() {
+        server.start();
+        return this;
+    }
+
+    @NotNull
+    public Http4kServer stop() {
+        if (server != null) server.stop();
+        return this;
+    }
+
+    @Override
+    public void block() {
+        try {
+            Thread.currentThread().join();
+        } catch (InterruptedException e) {
+            throw new RuntimeException(e);
```

Comment:
Just out of curiosity, does http4k handle this runtime exception properly? or should developers be careful about shutting down an app? As you're the creator of http4k, I believe this is the best way but I just wanted to understand what this method is.

Reply:
That exception won't actually ever occur in the normal running of an App. It's actually only ever used at the top level (it's part of the outermost Server API), and is a bit of an integration niggle due to checked/unchecked exceptions.

Reply:
That makes sense. Thanks!